### DISCUSSION TITLE
Инструкция

### DISCUSSION THREAD

**Discussion by nexonx** [Tseka85/opencart3-exchange1c#42] [General]
Добрый день! Где можно скачать пошаговую инструкцию по установке модуля на  ocStore-3.0.3.7 ?

**Comment by Tseka85** [comment]
К сожалению , такой инструкции не существует.

**Reply by nexonx** [reply]
Установил opencart3-exchange1c-v1.6.4.8(not official).ocmod.zip   Как пользоваться модулем ?

**Reply by Tseka85** [reply]
Зайдите в настройки модуля и всё станет понятно. Сделайте настройки под свою выгрузку .

**Reply by nexonx** [reply]
Папки на export, bitrix, admin  нужно на закачать на сайт?

**Reply by Tseka85** [reply]
Да .

**Reply by nexonx** [reply]
Я правильно понял, что модуль использует встроенный механизм 1C "Обмен данными с WEB-сайтом"  который для битрикса предназначен?

**Reply by Tseka85** [reply]
Да.

**Reply by nexonx** [reply]
Где посмотреть коды ошибок? 
Выгружено товаров: 9
Выгружено картинок: 0
Выгружено файлов: 0
Выгружено предложений: 7
Получен расширенный статус успешного завершения сеанса
Ответ сервера:
Successfully import catalog 

Получен расширенный статус успешного завершения сеанса
Ответ сервера:
Successfully import catalog 

Произошла ошибка на стороне сервера. Получен неизвестный статус импорта.
Обмен не выполнен
Ответ сервера:
ERROR: 3000

Выгрузка товаров завершена с ошибками!!!

**Reply by nexonx** [reply]
3000 | ОШИБКА! Файл не является стандартом XML или обрезан
  Что это значит?
Что делать?

**Reply by Tseka85** [reply]
Версия 1С ?

**Comment by nexonx** [comment]
Управление торговлей", редакция 10.3 (10.3.73.1)

**Comment by Tseka85** [comment]
А версия самой платформы 1С ?

**Reply by nexonx** [reply]
1С:Предприятие 8.3 (8.3.20.1674)

**Reply by Tseka85** [reply]
У вас чуть ли не последняя версия 1С . Надо смотреть логи . Позже скину порядок настроек модуля опенкарт . 

Можете мне скинуть кусок XML выгрузки из 1С ?

**Reply by nexonx** [reply]
Могу прислать, но я обычный системный администратор, мне нужна инструкция где взять "кусок XML выгрузки из 1С"

**Comment by Tseka85** [comment]
В 1С где вы вставляли ссылку вашего сайта , вместо выгрузки на сайт , укажите выгрузка в каталог .

**Reply by nexonx** [reply]
https://disk.yandex.ru/d/zHBjmtSd-5Oa3w

**Comment by Fixloop** [comment]
привет всем, у меня "3000 ОШИБКА! Файл не является стандартом XML или обрезан" более 2 месяцев назад я как то решил эту проблема, сейчас не получается, помогите, название и версия конфигурации  "Управление торговлей", редакция 10.3 (10.3.47.3), платформа 1С:Предприятие 8.3 (8.3.18.1289), мой телеграмм @Fixloop - если можно напишите что делать, разные варианты попробовал но безуспешно, точно знаю что должно работать, так как до этого я настраивал плагин и он работал, очень прошу помочь в решении проблемы

**Comment by andqzzz** [comment]
Добрый день! Установил, проверил - папка \system\storage\cache\exchange1 на месте, \storage\cache\exchange1 тоже есть, для записи открыты, но при загрузке ошибка 3001. После третьей переустановки магазина ручная загрузка типа сработала, но товар не появился. После включения 

HTTPS снова ошибка 3001.

Кроме того, после включения модуля, в самом модуле написано "включено", а в списке модулей всё равно "выключен".

Помогите пожалуйста разобраться.

**Comment by Fixloop** [comment]
Здравствуйте, почему вы мне пишите об этом? Лично я разобрался с плагином, были проблемы и те что у вас тоже, я не создатель этого плагина, я просто с ним разобрался и настроил обмен

Best wishes, Max B.

2 авг. 2023 г., в 20:24, andqzzz ***@***.***> написал(а):

﻿

Добрый день! Установил, проверил - папка \system\storage\cache\exchange1 на месте, \storage\cache\exchange1 тоже есть, для записи открыты, но при загрузке ошибка 3001. После третьей переустановки магазина ручная загрузка типа сработала, но товар не появился. После включения

HTTPS снова ошибка 3001.

Кроме того, после включения модуля, в самом модуле написано "включено", а в списке модулей всё равно "выключен".

Помогите пожалуйста разобраться.
